tools/update-metadata/src/lib.rs

```diff
@@ -448,7 +448,17 @@ fn find_migrations_forward(
 #[cfg(test)]
 mod tests {
     use super::*;
-    use chrono::{DateTime, Duration, NaiveDate, Utc};
+    use chrono::{DateTime, Duration, NaiveDate, TimeDelta, Utc};
+
+    // Deal with a deprecated function and avoid unwrapping everywhere causing line wraps.
+    fn milliseconds(milliseconds: i64) -> TimeDelta {
+        Duration::try_milliseconds(milliseconds).unwrap()
+    }
```

Comment:
What function is deprecated? chrono's [Duration::milliseconds](https://docs.rs/chrono/latest/chrono/type.Duration.html#method.milliseconds) doesn't seem to be marked that way.

Reply:
I don't really know why we're getting the warnings if that's the case. Maybe they have some conditionally different version of the library with the nightly compiler? Check out the spew in Patrick's PR. This makes it go away https://github.com/bottlerocket-os/twoliter/actions/runs/9102587464/job/25022559043

Reply:
Hmmm: maybe the un-deprecated it https://docs.rs/chrono/0.4.35/chrono/type.Duration.html